### DISCUSSION TITLE
PTP Meetup! #1

### DISCUSSION THREAD

**Discussion by shotaro8yagi** [project-theory-probe/ptp-voyage#8] [General]
12月7日（木）に開催するPTP Meetup! #1 についての議論

**Comment by shotaro8yagi** [comment]
- カバー画像
- 趣旨
- 開催概要（場所・開催時間・アクセス・無料・持ち込み歓迎）
- お品書き（連載トーク・プロトタイプの紹介＝記事にまだなってないもの）
- LP的な場所
- 申込フォーム（sender）でやってみる

**Comment by kikuchiharuma** [comment]
LP的なページ
作ってみました
エントリーフォームも試しに置いています
https://deploy-preview-9--soft-custard-acf2d5.netlify.app/meetup/1/

**Reply by kikuchiharuma** [reply]
@shotaro8yagi エントリーするとお礼のメールが返ってくると思うので、試してみていただけませんか

**Reply by shotaro8yagi** [reply]
登録してみました！お礼メールまで、スムーズです！
![image](https://github.com/project-theory-probe/ptp-voyage/assets/49149959/424a087f-92cc-45b6-81db-de20baf7c1e0)
ただ、このオレンジのmeetup#1ボタン押すとPage not Foundが出てきちゃいました

**Reply by kikuchiharuma** [reply]
ありがとうございます！
イベントページは公開前なので Page not Found で問題ないです

**Comment by shotaro8yagi** [comment]
基本情報作りました！

> 趣旨
> 開催概要（場所・開催時間・アクセス・無料・持ち込み歓迎）
> お品書き（連載トーク・プロトタイプの紹介＝記事にまだなってないもの）

https://docs.google.com/document/d/1A4Z44rosC-_dfgVIeZDOlF4sxN_zQJAJWx5qxdbpnzA/edit

画像（と四コマ）鋭意作成中です…プロジェクトおじさんに乾杯させる予定

**Reply by kikuchiharuma** [reply]
ありがとうございます！

**Reply by kikuchiharuma** [reply]
反映しました！
https://deploy-preview-9--soft-custard-acf2d5.netlify.app/meetup/1/

**Reply by kikuchiharuma** [reply]
@shotaro8yagi アジェンダはもう一押し具体的に、こんな感じでどうでしょう

- 連載「自己組織化するプロジェクトを見たい」トーク 八木翔太郎
- 連載「ハッカー文化の伝道師」トーク 菊地玄摩
- プロジェクト探査プロトタイプ Value People 紹介 菊地玄摩
- Project Theory Probe 座談会
- 懇親会

最初ご挨拶的なものはあるかも知れないですが、書かなくてもよいかも？

**Reply by kikuchiharuma** [reply]
他はこの状態で公開してしまっても良いのかなと思いました。
宣伝を開始したいですね

**Reply by shotaro8yagi** [reply]
@kikuchiharuma 
レス遅れてすみません。アジェンダ良いと思いました！（有難うございます！）

> 連載「自己組織化するプロジェクトを見たい」トーク 八木翔太郎
> 連載「ハッカー文化の伝道師」トーク 菊地玄摩
> プロジェクト探査プロトタイプ Value People 紹介 菊地玄摩
> Project Theory Probe 座談会
> 懇親会

あとは、先日pilotboat共有会で定金さんとも確認したのですが、以下太字を追記したいです。２点目はこころさんのような方の参加を想定しています。senderのアドレス宛に連絡していただくか、予めフォームにチェックボックスを入れておくかどちらかかなと思いました。

> 形式
> 前半は座談会形式。後半は質疑を交えつつ、交流タイムを設けます。途中参加・途中抜け歓迎です。
> **※会場の都合により、20時以降は内側からの開錠となりますので遅れて到着の場合にはビル入口にて、070-3168-6870（八木）までお電話下さい
> ※遠方などの理由によりオンライン参加希望の方は[[連絡先アドレス]]までDM下さい**
>

**Reply by shotaro8yagi** [reply]
あと、告知文ですが「ジャーナル読んで面白いと思った方は…」という紹介文にしてptp.voyageのリンク付けるのが良いのでは、という話になりました。そうすればイベントの位置づけや立場とかの言及はしないで済むので。

**Reply by kikuchiharuma** [reply]
@shotaro8yagi 会場の案内、アジェンダのアップデート反映しました。何カ所か表現を変えたり、追加した内容があります。確認してほしいです。https://deploy-preview-9--soft-custard-acf2d5.netlify.app/meetup/1/

オンライン参加については、特に隠す必要もないので、リンクを置いておけば良いと思ったのですがどうでしょう。
Google Meet のイベントを作って置いてみました。

告知文は熱い感じが出ていていいと思ったので、内容はそのままリンクを追加しています。
アジェンダからもリンクしたので、Journal とのつながりはわかりやすくなったのではないかと思います。

**Reply by kikuchiharuma** [reply]
それから、フォームの説明が必須ぽく見えたので、アップデートの配信を希望する場合はエントリーしてね、というテキストに変えました。

**Reply by kikuchiharuma** [reply]
また、# は HTML だとエスケープが必要だったり、SNS ではタグ扱いになってしまったりと具合が悪いので、単にタイトルに Dec. 7 2023 入れる形にしてみました。

**Comment by shotaro8yagi** [comment]
挿絵的に使えたら
![プロジェクトおじさん乾杯](https://github.com/project-theory-probe/ptp-voyage/assets/49149959/67eb6942-7d15-4f88-8b2a-e01932fde67c)

**Reply by kikuchiharuma** [reply]
反映しました！
og image にしたので、SNS でイベントページのリンクをシェアすると表示されると思います
